src/main/java/entity/Conversation.java

```diff
@@ -0,0 +1,35 @@
+package entity;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class Conversation {
```

Comment:
is the messaging class going to be in a distinct panel from the rest?

Reply:
Probably, yes, if we even get around to implementing it